nwbinspector/checks/tables.py

```diff
@@ -92,15 +94,18 @@ def check_column_binary_capability(table: DynamicTable, nelems: int = 200):
         if hasattr(column, "data") and not isinstance(column, VectorIndex):
             if np.asarray(column.data[0]).itemsize == 1:
                 continue  # already boolean, int8, or uint8
-            unique_values = np.unique(column.data[:nelems])
+            try:
+                unique_values = np.unique(column.data[:nelems])
+            except Exception:  # some columns can contain references to other objects in the NWBFile
+                continue
```

Comment:
Is there a better way to catch this? I'd rather not use try/except, but if we do can we make the exception type more specific?

Reply:
Yep, I can catch the issue with a`TypeError`. It occurs whenever the contained object has no defined basis for comparison like a ">" operator (which `np.unique` uses). Likewise, the same object is unable to be hashed so something like `np.array(set(...))` also throws a `TypeError`, so either way I think we do need this.

The same underlying problem was triggered by two cases, one where the actual raw column data was a `np.array` with `dtype="O"`/`object`, and the other was when the column data was `ElectrodeGroup` references.

Reply:
While we could program explicit `if` skips for those specific cases instead of the `try/except`, I feel like that's a bandaid and we'd ultimately end  up having to add more and more `if` checks for other items that can't be passed through `np.uniques` or `set`.

Reply:
what about something like `if hasattr(data, '__gt__`)`

Reply:
Nope🤨 the offending data objects have that defined as an attribute, but the attribute is a method that does not support inter-object comparison, and further raises a `NotImplementedError` if you really try to force it to work by setting its comparison value.

Reply:
hmm ok